test/node-unit/cli/config.spec.js

```diff
@@ -52,7 +62,7 @@ describe('cli/config', function() {
         it('should use the JS parser', function() {
```

Comment:
A test for the `.cjs` extension is missing.

Reply:
ah, didn't notice!  just added one.

Reply:
still missing